modules/pam_unix/unix_chkpwd.c

```diff
@@ -137,7 +137,7 @@ int main(int argc, char *argv[])
 	  user = getuidname(getuid());
 	  /* if the caller specifies the username, verify that user
 	     matches it */
-	  if (strcmp(user, argv[1])) {
+	  if (!user || strcmp(user, argv[1])) {
```

Comment:
Could you please change that to `user == NULL`

Reply:
Done